engines/saga2/localize.h

```diff
@@ -66,8 +66,8 @@ namespace Saga2 {
 #define PHILLIP_BROSTATE    "Phillip is: "
 #define KEVIN_BROSTATE      "Kevin is: "
 
-#define ON_AGRESS           "Agression: On"
-#define OFF_AGRESS          "Agression: Off"
+#define ON_AGRESS           "Aggression: On"
+#define OFF_AGRESS          "Aggression: Off"
```

Comment:
In-game data. Can it be verified that the original was like this?

Reply:
Same as above

Reply:
Yes, it is how the original was. Please, remove this change

Reply:
@sev- : This PR was closed without being merged, so this change is not present in master.

It was replaced by PR #4438 which I had edited so that only fixes to comments were present. All fixes to "game data" were replaced by comments that the spelling was incorrect, but changing it would affect game data.